website/docs/cloud-docs/workspaces/variables/managing-variables.mdx

```diff
@@ -123,7 +123,7 @@ To create a variable set:
 
 To edit or remove a variable set:
 
-1. Click **Settings** in the top menu bar and then click **Variable Sets** in the left sidebar. The **Variable sets** page appears.
+1. Click **Settings**, then click **Variable Sets**. The **Variable sets** page appears.
```

Comment:
I like how you've done this! But I'm wondering --> do you think it'd be clearer for us to say:

Go to the organization settings and then click **Variable Sets**. 

I'm just wondering if we need to specify that it's the organization settings, or if we think this is clear enough.